src/main/java/com/rk/rentacarapp/security/SecurityConfigurer.java

```diff
@@ -0,0 +1,59 @@
+package com.rk.rentacarapp.security;
+
+import com.rk.rentacarapp.filters.JwtRequestFilter;
+import com.rk.rentacarapp.services.MyUserDetailsService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.crypto.password.NoOpPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+
+/*
+    * JWT configured based on this guide: https://www.youtube.com/watch?v=X80nJ5T7YpE
+ */
+
+@EnableWebSecurity
+@EnableGlobalMethodSecurity(
+        prePostEnabled = true,
+        securedEnabled = true,
+        jsr250Enabled = true)
+public class SecurityConfigurer extends WebSecurityConfigurerAdapter {
+    @Autowired
+    private MyUserDetailsService myUserDetailsService;
```

Comment:
Variable name shouldn't start with my as well, this could just be a userDetailsService